Backend/contents.js

```diff
@@ -0,0 +1,44 @@
+const express = require("express");
+const app = express();
+const cors = require("cors");
+const accountsDb = require("./db/accounts");
+app.use(cors());
+
+const accounts = [
```

Comment:
Add one ENV variable in backend. If that variable is True then use Mock data if not then use DB data